### DISCUSSION TITLE
请问markdown写什么，会展示成infobox的形态

### DISCUSSION THREAD

**Discussion by BEWINDOWEB** [ppoffice/hexo-theme-icarus#1316] [Q&A] [answered]
![截屏2024-08-22 21 25 44](https://github.com/user-attachments/assets/2e34aec2-3118-4501-8850-c82f6a0348d3)

想要实现类似这种展示效果，请问对应的markdown应该怎么写？🙏

**Comment by BEWINDOWEB** [comment]
我看到 https://raw.githubusercontent.com/ppoffice/hexo-theme-icarus/site/source/_posts/zh-CN/Widgets.md 写的是：
```
<article class="message message-immersive is-danger">
  <div class="message-body">
    <i class="fas fa-exclamation-triangle mr-2"></i><a href="https://support.google.com/feedburner/answer/10483501" target="_blank">Google即将关闭Feedburner的邮件订阅功能。</a> 你可以切换到<a href="#follow-it">follow.it挂件</a>或者其他邮件订阅服务。
  </div>
</article>
```
只能这样手写HTML是吗？有没有更优雅的方式呢 @ppoffice 🙏

**Reply by ppoffice** [reply, marked as answer]
这种只能是HTML。你可以试试另一种info box：https://ppoffice.github.io/hexo-theme-icarus/uncategorized/%E8%87%AA%E5%AE%9A%E4%B9%89hexo%E6%A0%87%E7%AD%BE%E6%8F%92%E4%BB%B6/

**Reply by BEWINDOWEB** [reply]
多谢
